jetty-deploy/src/main/java/org/eclipse/jetty/deploy/bindings/StandardUndeployer.java

```diff
@@ -44,31 +42,6 @@ public void processBinding(Node node, App app) throws Exception
     {
         ContextHandler handler = app.getContextHandler();
         ContextHandlerCollection chcoll = app.getDeploymentManager().getContexts();
-
-        recursiveRemoveContext(chcoll,handler);
-    }
-
-    private void recursiveRemoveContext(HandlerCollection coll, ContextHandler context)
-    {
-        Handler children[] = coll.getHandlers();
-        int originalCount = children.length;
-
-        for (int i = 0, n = children.length; i < n; i++)
-        {
-            Handler child = children[i];
-            LOG.debug("Child handler {}",child);
-            if (child.equals(context))
-            {
-                LOG.debug("Removing handler {}",child);
-                coll.removeHandler(child);
-                child.destroy();
-                if (LOG.isDebugEnabled())
-                    LOG.debug("After removal: {} (originally {})",coll.getHandlers().length,originalCount);
-            }
-            else if (child instanceof HandlerCollection)
-            {
-                recursiveRemoveContext((HandlerCollection)child,context);
-            }
-        }
+        chcoll.undeployHandler(handler);
```

Comment:
@joakime I have removed the recursiveRemoveContext - because I could not think of why it was being done (not symmetric with the StandardDeployer).  But I suspect I'm missing something??? Do you recall why this was needed?